include/openmc/distribution.h

```diff
@@ -80,6 +80,29 @@ class Uniform : public Distribution {
   double b_; //!< Upper bound of distribution
 };
 
+//==============================================================================
+//! Rational distribution over the interval [a,b] with exponent n
+//==============================================================================
+
+class Rational : public Distribution {
```

Comment:
Rather than calling this distribution `Rational`, I'd prefer to name it `PowerLaw`. Can you make changes accordingly throughout?